wordfreq/build.py

```diff
@@ -131,6 +133,24 @@ def get_db_connection(filename):
     return sqlite3.connect(filename)
 
 
+def read_leeds_wordlist_into_db(conn, filename, dbname, lang):
+    logger.info("Loading %r" % filename)
+    wordlist = read_leeds_corpus(filename)
+    save_wordlist_to_db(conn, dbname, lang, wordlist)
+
+
+def read_wordlist_into_db(conn, filename, dbname, lang='*'):
+    logger.info("Loading %r", filename)
+    if lang == '*':
+        multi_wordlist = read_multilingual_csv(filename)
+        for lang in multi_wordlist:
+            logger.info("\tLanguage: %s", lang)
+            save_wordlist_to_db(conn, dbname, lang, multi_wordlist[lang])
```

Comment:
It is somewhat confusing that the variable name `lang` that was an argument gets reused as something else in this loop.